server/src/entity/Transaction.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinTable,
+  ManyToMany,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  PrimaryColumn,
+  OneToOne,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Account } from './Account';
+import { Visit } from './Visit';
+
+@Entity()
+export class Transaction extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @OneToOne((type) => Visit, (visit) => visit.id)
```

Comment:
See above comments about `OneToOne`

Reply:
Removed due to the fact that a visit entity was already an attribute.